tests/unit/test_pep425tags.py

```diff
@@ -159,3 +159,22 @@ def test_manylinux1_tag_is_first(self):
                 assert arches == ['manylinux1_x86_64', 'linux_x86_64', 'any']
             else:
                 assert arches == ['manylinux1_x86_64', 'linux_x86_64']
+
+    def test_manylinux1_check_glibc_version(self):
```

Comment:
All the `two_twenty` cases you test are regular that are expected to pass against some normal major / minor. However no testing of unexpected input, like an empty string, or one containing no period, etc.